<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>optionwindow</class>
 <widget class="QDialog" name="optionwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>361</width>
     <height>491</height>
    </rect>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>9</y>
      <width>341</width>
      <height>431</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="MeshSetupTab">
     <attribute name="title">
      <string>Mesh Setup</string>
     </attribute>
     <widget class="QFrame" name="spaceFrame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>311</width>
        <height>91</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QSpinBox" name="zDimBox">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>60</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="zLabel">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>40</y>
         <width>91</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>Z-Dimension</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="yDimBox">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>60</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="yLabel">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>40</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Y-Dimension</string>
       </property>
      </widget>
      <widget class="QLabel" name="xLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>91</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>X-Dimension</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="xDimBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="wrapping">
        <bool>false</bool>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="suffix">
        <string/>
       </property>
       <property name="prefix">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="gridSizeLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string>Spatial  Set up</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="timeFrame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>311</width>
        <height>91</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="timeLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>81</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>Time Set up</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="stepNumberBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="timeStepBox">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>60</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="suffix">
        <string>e-10</string>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="stepLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>121</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>Number of Steps</string>
       </property>
      </widget>
      <widget class="QLabel" name="timeStepLabel">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>40</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Time Step</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="speedLight">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>10</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="decimals">
        <number>10</number>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="ExternalFieldFrame">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>210</y>
        <width>151</width>
        <height>171</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="externalLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>151</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>External B-Field</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="ExBxSpinBox">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>50</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="suffix">
        <string>T</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="ExBySpinBox">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>90</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="suffix">
        <string>T</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="ExBzSpinBox">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>130</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="suffix">
        <string>T</string>
       </property>
      </widget>
      <widget class="QLabel" name="ExBxLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>21</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>Bx</string>
       </property>
      </widget>
      <widget class="QLabel" name="ExByLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>21</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>By</string>
       </property>
      </widget>
      <widget class="QLabel" name="ExBzLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>21</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>Bz</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="boundaryFrame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>151</width>
        <height>81</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="boundaryLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>151</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>Boundary Set-up</string>
       </property>
      </widget>
      <widget class="QComboBox" name="boundaryChoice">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>131</width>
         <height>37</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>ABC</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QFrame" name="algorithmFrame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>151</width>
        <height>80</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="algorithmLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>131</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>Position Algorithm</string>
       </property>
      </widget>
      <widget class="QComboBox" name="algorithmBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Taylor </string>
        </property>
       </item>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="SourcesTab">
     <attribute name="title">
      <string>Sources Setup</string>
     </attribute>
     <widget class="QComboBox" name="SourceBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>7</y>
        <width>126</width>
        <height>37</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Particle</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ricker</string>
       </property>
      </item>
     </widget>
     <widget class="QFrame" name="positionFrame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>311</width>
        <height>91</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QSpinBox" name="zpositionBox">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>60</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="zpositionLabel">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>40</y>
         <width>71</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>Z Position</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="ypositionBox">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>60</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="ypositionLabel">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>40</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Y Position</string>
       </property>
      </widget>
      <widget class="QLabel" name="xpositionLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>71</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>X Position</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="xpositionBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="wrapping">
        <bool>false</bool>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="suffix">
        <string/>
       </property>
       <property name="prefix">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="positionLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string>Position  Set up</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="velocityFrame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>311</width>
        <height>91</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QDoubleSpinBox" name="zvelocityBox">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>60</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>91</width>
         <height>21</height>
        </size>
       </property>
       <property name="suffix">
        <string>c</string>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="zvelocityLabel">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>40</y>
         <width>71</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>Z Velocity</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="yvelocityBox">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>60</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="suffix">
        <string>c</string>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="yaccelerationLabel">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>40</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Y Velocity</string>
       </property>
      </widget>
      <widget class="QLabel" name="xvelocityLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>71</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>X Velocity</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="xvelocityBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="wrapping">
        <bool>false</bool>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="suffix">
        <string>c</string>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="velocityLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string>Velocity Set up</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="accelerationFrame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>311</width>
        <height>91</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QDoubleSpinBox" name="zaccelerationBox_2">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>60</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>91</width>
         <height>21</height>
        </size>
       </property>
       <property name="suffix">
        <string/>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="zaccelerationLabel_2">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>40</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Z-Accel</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="yaccelerationBox">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>60</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="suffix">
        <string/>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="yaccelerationLabel_2">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>40</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Y-Accel</string>
       </property>
      </widget>
      <widget class="QLabel" name="xaccelerationLabel_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>X-Accel</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="xaccelerationBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="wrapping">
        <bool>false</bool>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="suffix">
        <string/>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="AccelerationLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string>Acceleration Set up</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="ppwFrame">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>3</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QDoubleSpinBox" name="ppwBox">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>10</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="ppwLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>41</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>ppw:</string>
       </property>
      </widget>
     </widget>
     <zorder>ppwFrame</zorder>
     <zorder>SourceBox</zorder>
     <zorder>positionFrame</zorder>
     <zorder>velocityFrame</zorder>
     <zorder>accelerationFrame</zorder>
    </widget>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>450</y>
      <width>104</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
